### DISCUSSION TITLE
GitHub Draft Pages

### DISCUSSION THREAD

**Discussion by ginkoid** [community/community#22079] [Other Feature Feedback, Questions, & Ideas]
<p>I noticed that some repositories have draft pages available at <code>*.drafts.github.io</code>. All of them seem to be in-development versions of otherwise published pages.</p>
<p><a href="https://www.google.com/search?q=site%3Adrafts.github.io" rel="nofollow noopener">A Google search reveals</a> that there are over 6000 currently published and indexed. I found <a href="https://github.community/t/10308">this post</a> which mentions that pages from non-default branches create drafts. However, I can’t figure out how to create a page draft of my own.</p>
<p>How would I create a GitHub pages draft?</p>
<p>Thanks!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/ginkoid">@ginkoid</a> - Welcome to the community! That is a feature of Jekyll, it lets you write draft posts, and allows you to easily preview how they will look prior to publishing.</p>
<p>To start a draft, create a folder called  <code>_drafts</code>  in your site’s source directory (e.g., alongside  <code>_posts</code> ), and add a new markdown file to it. To preview your new post, run the  <code>jekyll serve</code>  command with the  <code>--drafts</code>  flag.</p>
<p>For more information about Jekyll and Pages. Check out our <a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#plugins">documentation</a>.</p>

**Reply by alxndr** [reply]
Is there a way to both publish the `--drafts` version of the site to GitHub Pages under the `*.drafts.github.io` (username-specific) subdomain, and also publish the normal build (not drafts) to GitHub Pages under the standard `*.github.io` (username-specific) subdomain?
